### PR TITLE
Info card rework

### PR DESCRIPTION
**Pull request by domikitchen**
## Description ##

Clearly documents the reason for the pull request and what has changed.
- [x] This is part of a feature.
    - This is a rework of an existing feature, currently not much has been changed, but it's functional.

## Organization ##

- [x] Have you linked this pull request to a Trello card?

## Review ##

- [x] Have you added two reviewers to this pull request?
- [x] Is this a work in process?
- [x] Is this ready to be reviewed?

### CODE REVIEW

File: src/components/pages/DataViz/bridge_card.js [hunk 1]
```diff
@@ -42,128 +40,109 @@ const BridgeCard = () => {
...
-                  image={detailsData.bridge_image}
-                />
+    <div className="detailsContainer">
+      <Card className={detailsData.root}>
+        <CardActionArea>
+          <div className="closeArea">
+            <div
+              className="close_button"
+              onKeyDown={e => {
+                console.log(e);
```
Comment (Vippsi) [inline src/components/pages/DataViz/bridge_card.js]: please remove the console.log

File: src/components/pages/DataViz/bridge_card.js [hunk 2]
```diff
@@ -42,128 +40,109 @@ const BridgeCard = () => {
...
+                  <span className="bottomInfoTags">Project Sub Stage:</span>{' '}
+                  {detailsData.sub_stage}
+                </p>
+                <p>
+                  <span className="bottomInfoTags">Project Stage:</span>{' '}
+                  {detailsData.project_stage}
+                </p>
+              </div>
+              <div className="bottom_info_cols">
+                {/* currently unsure if following info could be implemented */}
```
Comment (Vippsi) [inline src/components/pages/DataViz/bridge_card.js]: Maybe consider adding this to Trello so we can keep the code clean and not forget about this in the future

**Review by Vippsi — CHANGES_REQUESTED** [review]
Everything looks good, just remove the console.log and take a look at the other suggestion

**Review by dortega5185 — APPROVED** [review]
Everything checks out. Good job on the info card rework Dominique.

**Review by jidelson — APPROVED** [review]
Looks good

**Pull request merged**